package.json

```diff
@@ -7,7 +7,8 @@
   "workspaces": [
     "packages/*",
     "examples/fa2",
-    "examples/hello-world"
+    "examples/hello-world",
+    "examples/get-tez"
```

Comment:
isn't it better to have access to the whole examples folder, including js ones?

Reply:
Not really, this is a set of globs that contain folders with `package.json`s. The JS examples don't have these.